dbms/src/Storages/System/StorageSystemParts.cpp

```diff
@@ -1,3 +1,4 @@
+#include <Storages/StorageLog.h>
```

Comment:
This is not the right one. Look at `MarkInCompressedFile.h`.
(You could consider the implementation of StorageMergeTree.)